### PR TITLE
RavenDB-7625 race in subscription test

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Client/Documents/TcpUtils.cs [hunk 1]
```diff
@@ -70,5 +69,12 @@ internal static class TcpUtils
             stream = sslStream;
             return stream;
         }
+
+        public static string GetTcpUrl(string host)
+        {
+            if (host.IndexOf(".fiddler", StringComparison.OrdinalIgnoreCase) > 0)
```
Comment (ayende) [inline src/Raven.Client/Documents/TcpUtils.cs:75]: Put this in `#if DEBUG`

Comment (fitzchak) [reply src/Raven.Client/Documents/TcpUtils.cs:75]: What if the user will supply a localhost.fiddler url in order to debug RavenDB?
In this case all tcp connections won't work.

Comment (ayende) [reply src/Raven.Client/Documents/TcpUtils.cs:75]: The user is using `ravendb.fiddler.com` ? That is a valid domain name.

Comment (ayende) [reply src/Raven.Client/Documents/TcpUtils.cs:75]: Better to actually handle that on the server side, instead.

Comment (fitzchak) [reply src/Raven.Client/Documents/TcpUtils.cs:75]: Fixed.

**Pull request merged**